proto/adaptors/aave/aave_v2_enable_asset_as_collateral_adaptor.proto

```diff
@@ -0,0 +1,41 @@
+/*
+* Protos for function calls to AaveV2EnableAssetAsCollateralAdaptor.sol
+*/
+
+syntax = "proto3";
+package steward.v3;
+
+option go_package = "/steward_proto";
+
+import "adaptors/base.proto";
+
+// Represents call data for the Aave AToken adaptor, used to manage lending positions on Aave
+message AaveV2EnableAssetAsCollateralAdaptorV1 {
+    oneof function {
+        /***** BASE ADAPTOR FUNCTIONS *****/
+
+        // Represents function `revokeApproval(ERC20 asset, address spender)`
+        RevokeApproval revoke_approval = 1;
```

Comment:
Is this actually necessary for this adapter?

Reply:
It's a base function for all protos that derive from BaseAdaptor.sol so I do it by default